stubs/parsimonious/parsimonious/nodes.pyi

```diff
@@ -26,15 +26,15 @@ class RegexNode(Node):
 
 class RuleDecoratorMeta(type): ...
 
-_VisitorResultT = TypeVar("_VisitorResultT")
+_VisitRT = Any
 
 class NodeVisitor(metaclass=RuleDecoratorMeta):
```

Comment:
Haven't looked at the code but I suspect this class should be made generic.

Reply:
Sorry, crazy week
You mean that `_VisitRT` should be `Generic` instead of `TypeAlias`?

Reply:
Oh, ok, think i figured it out
Like that?

```python
_VisitorResultT = TypeVar("_VisitorResultT")

class NodeVisitor(Generic[_VisitorResultT], metaclass=RuleDecoratorMeta):
    ...
    def visit(self, node: Node) -> _VisitorResultT: ...
    ...
```